config/jobs/kubernetes-sigs/kwok/kwok-presubmits-main.yaml

```diff
@@ -55,6 +58,7 @@ presubmits:
   - name: pull-kwok-e2e-test-main
     decorate: true
     path_alias: sigs.k8s.io/kwok
+    cluster: eks-prow-build-cluster
```

Comment:
``` yaml
          requests:
            memory: "1Gi"
            cpu: "1"
          limits:
            memory: "2Gi"
            cpu: "2"
```

I'm not sure of the exact request but I think it should be enough, if not I'll add more 😄.

Reply:
One other point of note, the `pull-test-infra-unit` requires guaranteed QoS so the requests will need to match the limits.

Reply:
Oh thanks

``` yaml
          requests:
            memory: "2Gi"
            cpu: "2"
          limits:
            memory: "2Gi"
            cpu: "2"
```

Reply:
@rjsadow After specifying the `resources` field, the [tests](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/29731/pull-test-infra-unit-test/1668709627019137024#1:build-log.txt%3A553-576) are not able to detect the  changes.